### PR TITLE
Remove launch icon and update BitwardenActionCard

### PR DESCRIPTION
**Pull request by david-livefront**
## 🎟️ Tracking

N/A

## 📔 Objective

This PR removes the `ic_launch` icon and replaces it with the `ic_external_link` icon.

Additionally, the `BitwardenActionCard` has been updated to better match designs.

## 📸 Screenshots

| Before | After |
| --- | --- |
|  |  |
|  |  |

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenActionCard.kt [hunk 1]
```diff
@@ -57,12 +56,9 @@ fun BitwardenActionCard(
...
     ) {
-        Column(
-            modifier = Modifier.padding(16.dp),
-        ) {
-            Row(
-                modifier = Modifier.fillMaxWidth(),
-            ) {
+        Row(modifier = Modifier.fillMaxWidth()) {
+            Spacer(Modifier.width(16.dp))
+            Row(modifier = Modifier.padding(top = 16.dp)) {
```
Comment (dseverns-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenActionCard.kt:61]: 💭 non-blocking but the use of the top padding here but the trailing spacer at the end of the card content seems disjointed. I am an expert in disjointed 😆 😢

**Review by dseverns-livefront — APPROVED** [review]
I am good with it as is, but also left minor thought

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenActionCard.kt (hunk 1 above)
Comment (david-livefront) [reply app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenActionCard.kt:61]: I agree and I was not super happy with how this turned out.

But I needed the close icon to not be padded while making sure everything else was.

**Comment by david-livefront** [comment]
Thanks @dseverns-livefront

**Pull request merged**